apps/music/js/app.js

```diff
@@ -44,18 +44,6 @@ if (navigator.mozSetMessageHandler) {
   navigator.mozSetMessageHandler('activity', activity => onActivity(activity));
 }
 
-var unknownArtist;
-var unknownTitle;
```

Comment:
Yes, you would need quite a few MP3's to see the scan progress bar in action. When 'scanProgress' fires with lots of new MP3's, it will happen repeatedly in rapid succession. I'm assuming this change shouldn't affect things though because presumably, the strings will get localized once and cached, right?